src/main/java/elevator/Elevator.java

```diff
@@ -0,0 +1,7 @@
+package elevator;
+
+public interface Elevator extends Comparable<Elevator>{
```

Comment:
@dwaynewade0033 lets also add following methods
1. Method to call elevator to specific floor.
2. Method to find which floor elevator is moving from/to if its direction is not `STOP`